lineage/Analyze.py

```diff
@@ -103,13 +103,14 @@ def Analyze_list(
     tHMMobj_list = [
         tHMM(X, num_states=num_states, **kwargs) for X in pop_list
     ]  # build the tHMM class with X
-    _, _, _, gammas, LL = fit_list(tHMMobj_list)
+    rng = kwargs.get('rng', 1)
+    _, _, _, gammas, LL = fit_list(tHMMobj_list, rng)
 
     for _ in range(5):
         tHMMobj_list2 = [
             tHMM(X, num_states=num_states, **kwargs) for X in pop_list
         ]  # build the tHMM class with X
-        _, _, _, gammas2, LL2 = fit_list(tHMMobj_list2)
+        _, _, _, gammas2, LL2 = fit_list(tHMMobj_list2, rng)
 
         if LL2 > LL:
             tHMMobj_list = tHMMobj_list2
```

Comment:
@aarmey In this function, we run the fitting multiple times and choose the best fit. If we have the random seed here, this multiple running would be meaningless, and also, if we don't set a random seed and run this fitting multiple times, then there is no guarantee it finds the same answers.. 
Should we have only 1-time fitting here?

Reply:
If you are passing the `rng` around, the seed will be different each fit, but the collection of them will be reproducible. e.g., the first time it'll be 1, second time 6, third time 23, etc.